official/docs/php/addresses/RetrieveAddress.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+\EasyPost\EasyPost::setApiKey("EASYPOST_API_KEY");
+
+$address = \EasyPost\Address::retrieve("adr_...");
+
+echo $addresss;
```

Comment:
```suggestion
echo $address;
```

My bad, here is another one